src/lib_json/CMakeLists.txt

```diff
@@ -180,7 +180,7 @@ if(BUILD_OBJECT_LIBS)
         OUTPUT_NAME jsoncpp
         VERSION ${PROJECT_VERSION}
         SOVERSION ${PROJECT_SOVERSION}
-        POSITION_INDEPENDENT_CODE ON
+        POSITION_INDEPENDENT_CODE ${BUILD_SHARED_LIBS}
```

Comment:
Does it really make sense? I would have remove it, that's all. I mean, BUILD_SHARED_LIBS is orthogonal to BUILD_OBJECT_LIBS.